### ISSUE TITLE
HTML

### ISSUE THREAD

**Issue by biancaletp** [biancaletp/azterea.github.io#1]
<!DOCTYPE html>
<html lang="pt-BR">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Azterea - Soluções Inteligentes</title>
    <link rel="stylesheet" href="style.css"> 
    <link rel="icon" href="favicon.ico" type="image/x-icon"> 
    
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;700&family=Orbitron:wght@400;600;700&display=swap" rel="stylesheet">
</head>
<body>
    <header>
        <nav class="navbar">
            <div class="container">
                <a href="#home" class="logo">Azterea</a>
                <ul class="nav-links">
                    <li><a href="#home">Home</a></li>
                    <li><a href="#services">Serviços</a></li>
                    <li><a href="#about">Sobre</a></li>
                    <li><a href="#feedback">Feedback</a></li>
                    <li><a href="#support">Suporte</a></li>
                    <li><a href="#contact">Contato</a></li>
                </ul>
                <div class="burger">
                    <div class="line1"></div>
                    <div class="line2"></div>
                    <div class="line3"></div>
                </div>
            </div>
        </nav>
    </header>

    <main>
        
        <section id="home" class="hero">
            <div class="hero-content container">
                <h1> No universo dos dados, a Azterea é a estrela-guia </h1>
                <p>Inovação e excelência em tecnologia para impulsionar o seu negócio ao próximo nível.</p>
                <a href="#services" class="btn-primary">Nossos Serviços</a>
            </div>
            
        </section>

        
        <section id="services" class="services section-padding">
            <div class="container">
                <h2>Nossos Serviços</h2>
                <p class="section-description">Soluções tecnológicas avançadas para o seu negócio.</p>
                <div class="services-grid">
                    <div class="service-card">
                        <i class="icon-data"></i> 
                        <h3>Dados</h3>
                        <ul>
                            <li>Análise e Business Intelligence (BI)</li>
                            <li>Engenharia de Dados</li>
                            <li>Data Warehouse/Data Lake</li>
                            <li>Armazenamento e Backup</li>
                            <li>Governança e Qualidade de Dados</li>
                        </ul>
                    </div>
                    <div class="service-card">
                        <i class="icon-ai"></i> 
                        <h3>IA</h3>
                        <ul>
                            <li>Machine Learning</li>
                            <li>Automação Inteligente (RPA + IA)</li>
                            <li>Análise Preditiva e Prescritiva</li>
                            <li>Processamento de Linguagem Natural (NLP)</li>
                            <li>Visão Computacional</li>
                        </ul>
                    </div>
                    <div class="service-card">
                        <i class="icon-cloud"></i> 
                        <h3>Cloud</h3>
                        <ul>
                            <li>Migração para Nuvem</li>
                            <li>Backup e Recuperação em Nuvem</li>
                            <li>Segurança em Nuvem</li>
                            <li>Armazenamento e Distribuição Global (Cloud Storage/CDN)</li>
                            <li>Big Data e Analytics</li>
                        </ul>
                    </div>
                </div>
            </div>
        </section>

        
        <section id="about" class="about section-padding">
            <div class="container">
                <h2>Sobre a Azterea</h2>
                <div class="about-content">
                    <p>A Azterea nasceu da paixão por tecnologia e do desejo de transformar o mundo dos negócios através de soluções inovadoras em Dados, IA e Cloud. Desde a nossa fundação, temos nos dedicado a ajudar empresas a desvendar o potencial de suas informações, otimizar processos e alcançar novos patamares de sucesso.</p>
                    <h3>Nossa Missão</h3>
                    <p>Capacitar nossos clientes com tecnologia de ponta, entregando soluções inteligentes e eficientes que geram valor real e sustentável.</p>
                    <h3>Nossa Visão</h3>
                    <p>Ser referência global em Dados, IA e Cloud, reconhecida pela excelência, inovação e pelo impacto positivo que geramos para a sociedade.</p>
                    <h3>Nossos Valores</h3>
                    <ul>
                        <li><strong>Inovação:</strong> Buscamos constantemente novas abordagens e tecnologias.</li>
                        <li><strong>Excelência:</strong> Comprometimento com a qualidade em tudo o que fazemos.</li>
                        <li><strong>Colaboração:</strong> Acreditamos no poder do trabalho em equipe e da parceria.</li>
                        <li><strong>Integridade:</strong> Agimos com ética, transparência e responsabilidade.</li>
                        <li><strong>Foco no Cliente:</strong> Nossos clientes são o centro de todas as nossas ações.</li>
                    </ul>
                    <h3>Nossa Equipe</h3>
                    <p>Contamos com uma equipe de especialistas altamente qualificados e apaixonados por tecnologia, prontos para enfrentar os desafios mais complexos e entregar resultados excepcionais.</p>
                    
                </div>
            </div>
        </section>

        
        <section id="feedback" class="feedback section-padding">
            <div class="container">
                <h2>O Que Nossos Clientes Dizem</h2>
                <p class="section-description">Confira alguns depoimentos e casos de sucesso que impulsionamos.</p>
                <div class="testimonials-slider">
                    
                    <div class="testimonial-card">
                        <p>"A Azterea revolucionou a forma como lidamos com nossos dados. A equipe é extremamente competente e os resultados foram surpreendentes!"</p>
                        <h4>João Silva</h4>
                        <span>CEO, Empresa X</span>
                    </div>
                    
                    <div class="testimonial-card">
                        <p>"Com a Azterea, conseguimos implementar soluções de IA que otimizaram nossos processos e reduziram custos significativamente."</p>
                        <h4>Maria Souza</h4>
                        <span>Diretora de Operações, Empresa Y</span>
                    </div>
                    
                </div>
                <div class="case-studies">
                    <h3>Estudos de Caso</h3>
                    <div class="case-study-card">
                        <h4>Projeto Alpha: Otimização de Logística com IA</h4>
                        <p>Descrição breve do caso de sucesso e os resultados alcançados.</p>
                        <a href="#" class="btn-secondary">Ver Detalhes</a>
                    </div>
                    <div class="case-study-card">
                        <h4>Projeto Beta: Migração Completa para a Nuvem</h4>
                        <p>Descrição breve do caso de sucesso e os resultados alcançados.</p>
                        <a href="#" class="btn-secondary">Ver Detalhes</a>
                    </div>
                    
                </div>
            </div>
        </section>

        
        <section id="support" class="support section-padding">
            <div class="container">
                <h2>Suporte</h2>
                <p class="section-description">Estamos aqui para ajudar!</p>
                <div class="support-content">
                    <div class="faq-section">
                        <h3>Perguntas Frequentes (FAQ)</h3>
                        <div class="faq-item">
                            <h4>Como posso solicitar um orçamento?</h4>
                            <p>Você pode solicitar um orçamento através do nosso formulário de contato na seção 'Contato' ou ligando para nós.</p>
                        </div>
                        <div class="faq-item">
                            <h4>Quais tecnologias vocês utilizam?</h4>
                            <p>Trabalhamos com as tecnologias mais recentes em Cloud (AWS, Azure, GCP), linguagens de programação (Python, R, Java), frameworks de ML (TensorFlow, PyTorch) e ferramentas de BI (Power BI, Tableau).</p>
                        </div>
                        
                    </div>
                    <div class="contact-channel">
                        <h3>Canais de Atendimento</h3>
                        <p>Se sua dúvida não foi respondida, entre em contato conosco:</p>
                        <ul>
                            <li><i class="icon-email"></i> Email: <a href="mailto:contato@azterea.com">contato@azterea.com</a></li>
                            <li><i class="icon-phone"></i> Telefone: <a href="tel:+5511999999999">(11) 99999-9999</a></li>
                            <li><i class="icon-chat"></i> Chat Online (em breve)</li>
                        </ul>
                    </div>
                </div>
            </div>
        </section>

        
        <section id="contact" class="contact section-padding">
            <div class="container">
                <h2>Entre em Contato</h2>
                <p class="section-description">Ficaremos felizes em conversar sobre suas necessidades.</p>
                <div class="contact-grid">
                    <div class="contact-form">
                        <form action="#" method="POST">
                            <div class="form-group">
                                <label for="name">Nome:</label>
                                <input type="text" id="name" name="name" required>
                            </div>
                            <div class="form-group">
                                <label for="email">Email:</label>
                                <input type="email" id="email" name="email" required>
                            </div>
                            <div class="form-group">
                                <label for="subject">Assunto:</label>
                                <input type="text" id="subject" name="subject">
                            </div>
                            <div class="form-group">
                                <label for="message">Mensagem:</label>
                                <textarea id="message" name="message" rows="5" required></textarea>
                            </div>
                            <button type="submit" class="btn-primary">Enviar Mensagem</button>
                        </form>
                    </div>
                    <div class="contact-info">
                        <h3>Informações de Contato</h3>
                        <p><i class="icon-location"></i> Endereço: Rua São Paulo, 2710 - Floresta, Joinville - SC</p>
                        <p><i class="icon-phone"></i> Telefone: (47) 98479-7516</p>
                        <p><i class="icon-email"></i> Email: <a href="mailto: contato@azterea.com"> contato@azterea.com</a></p>
                        <div class="social-links">
                            <a href="#" target="_blank"><i class="icon-linkedin"></i></a>
                            <a href="#" target="_blank"><i class="icon-twitter"></i></a>
                            <a href="#" target="_blank"><i class="icon-facebook"></i></a>
                            <a href="#" target="_blank"><i class="icon-instagram"></i></a>
                        </div>
                    </div>
                </div>
            </div>
        </section>
    </main>

    <footer>
        <div class="container">
            <p>&copy; 2025 Azterea. Todos os direitos reservados.</p>
            <p>Desenvolvido pela Azterea.</p>
        </div>
    </footer>

    <script src="script.js"></script> 
</body>
</html>
